README.md

```diff
@@ -2,80 +2,40 @@
-| `infraToken`                     | Sematext Infra token              | `Nil` Provide your Infra token            |
-| `region`                         | Sematext region                   | `US` Sematext US or EU region             |
-| `agent.image.repository`         | The image repository              | `sematext/agent`                          |
-| `agent.image.tag`                | The image tag                     | `latest`                                  |
-| `agent.image.pullPolicy`         | Image pull policy                 | `Always`                                  |
-| `agent.service.port`             | Service port                      | `80`                                      |
-| `agent.service.type`             | Service type                      | `ClusterIP`                               |
-| `agent.resources`                | Agent resources                   | `{}`                                      |
-| `logagent.image.repository`      | The image repository              | `sematext/logagent`                       |
-| `logagent.image.tag`             | The image tag                     | `latest`                                  |
-| `logagent.image.pullPolicy`      | Image pull policy                 | `Always`                                  |
-| `logagent.resources`             | Logagent resources                | `{}`                                      |
-| `logagent.extraHostVolumeMounts` | Extra mounts                      | `{}`                                      |
-| `serverBaseUrl`                  | Custom Base URL                   | `Nil`                                     |
-| `logsReceiverUrl`                | Custom Logs receiver URL          | `Nil`                                     |
-| `eventsRecieverUrl`              | Custom Event receiver URL         | `Nil`                                     |
-| `serviceAccount.create`          | Create a service account          | `true`                                    |
-| `serviceAccount.name`            | Service account name              | `Nil` Defaults to chart name              |
-| `rbac.create`                    | RBAC enabled                      | `true`                                    |
-| `tolerations`                    | Tolerations                       | `[]`                                      |
-| `nodeSelector`                   | Node selector                     | `{}`                                      |
-
-For example, if you want to deploy Sematext Agent in EU region, create the following resource:
+Once installed, you can create `SematextAgent` resource that deploys Sematext Agent to all nodes in your cluster via `DaemonSet` resource:
 
 ```yaml
 apiVersion: sematext.com/v1alpha1
 kind: SematextAgent
 metadata:
   name: test-sematextagent
 spec:
-  region: "EU"
-  containerToken: YOUR_CONTAINER_TOKEN
```

Comment:
I would test this before removing it, not sure how it works the operator or which token use

Reply:
The operator uses the Helm chart, so it has the same parameters. This token does not exist anymore